internal/encoding/merge/parquet_test.go

```diff
@@ -0,0 +1,202 @@
+package merge
+
+import (
```

Comment:
format imports

Reply:
@atris could you double-check all imports in these files? They're not properly formatted.  You should follow the existing structure where
```go
import (
   <standard libraries>
   <empty line>
   <non-standard libraries>
)
```


For example here, it should be:
```go
import (
	"bytes"
	"testing"
	
	goparquet "github.com/fraugster/parquet-go"
	"github.com/fraugster/parquet-go/parquet"
	"github.com/kelindar/talaria/internal/encoding/block"
	"github.com/kelindar/talaria/internal/encoding/typeof"
	"github.com/stretchr/testify/assert"
)
```